contrib/depends/packages/boost.mk

```diff
@@ -3,13 +3,15 @@ $(package)_version=1_64_0
 $(package)_download_path=https://dl.bintray.com/boostorg/release/1.64.0/source/
 $(package)_file_name=$(package)_$($(package)_version).tar.bz2
 $(package)_sha256_hash=7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+$(package)_dependencies=libiconv
```

Comment:
boost locale won't build without libiconv or icu. Needed to make sure libiconv gets built before boost.